developer/src/tike/project/Keyman.Developer.System.Project.ProjectLog.pas

```diff
@@ -38,44 +38,18 @@ interface
 
 const
   ProjectLogStateTitle: array[TProjectLogState] of string = (
-    '',
-    'Hint',
-    'Warning',
-    'Error',
-    'Fatal Error',
-    'Success',
-    'Failure'
+    'info',
+    'hint',
+    'warn',
+    'error',
+    'fatal',
+    'info',
+    'info'
```

Comment:
Does this still correspond to l. 31 (plsFailure)

Reply:
Yes, the build result messages are 'info' level messages now, but have special formatting. See kmc's version:

https://github.com/keymanapp/keyman/blob/0ebc5d6aab55c62863cc4dda878d01ce8900b65b/developer/src/kmc/src/util/NodeCompilerCallbacks.ts#L216-L228